Examples/UIComponentExample/Examples/Complex layout/Components/UserProfile.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  UserProfile.swift
+//  UIComponentExample
+//
+//  Created by y H on 2021/7/21.
+//  Copyright © 2021 Luke Zhao. All rights reserved.
+//
+
+import UIComponent
+import Kingfisher
+import UIKit.UIImageView
+
+extension ViewComponent {
+  fileprivate func shadowAvatar() -> ViewUpdateComponent<Self> {
+    update{
+      $0.layer.shadowColor = UIColor.black.withAlphaComponent(0.2).cgColor
+      $0.layer.shadowOffset = CGSize(width: 0, height: 3)
+      $0.layer.shadowRadius = 6
+      $0.layer.shadowOpacity = 1
+    }
+  }
+}
+
+struct UserProfile: ComponentBuilder {
+  
+  let avatar: String
+  let userName: String
+  let introduce: String
+  let gender: String
```

Comment:
remove `gender`. Don't think we should show `gender` in our example.

Reply:
✅